<ui version="4.0" >
 <class>BtConfigWidget</class>
 <widget class="QWidget" name="BtConfigWidget" >
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gFolderList" >
     <property name="title" >
      <string>Search Folders</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Please add the source folders in which to search for the files in the backtrace:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KUrlRequester" name="edtUrl" />
      </item>
      <item>
       <widget class="QListWidget" name="lstFolders" >
        <property name="whatsThis" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Insert the source folders here. As example, for KDE development you have to add the Qt and Frameworks source folders:&lt;/p>
&lt;ul style="-qt-list-indent: 1;">&lt;li style=" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">/path/to/kdelibs&lt;/li>
&lt;li style=" margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">/path/to/qt/src&lt;/li>&lt;/ul>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>218</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemove" >
          <property name="text" >
           <string>&amp;Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdd" >
          <property name="text" >
           <string>&amp;Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>File types:</string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtExtensions" >
          <property name="whatsThis" >
           <string>List of file extensions. Example: *.cpp, *.h, *.c, etc...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
